cars.js

```diff
@@ -103,33 +107,48 @@ function generateSingleCar(index) {
     DOM.cardsData = document.getElementById("data-cards");
     DOM.tableData = document.getElementById("table-data");
     DOM.tableHead = document.getElementById("table-head");
-    DOM.whatToDraw = "list"
+    DOM.whatToDraw = "list";
 
     draw(cars, DOM.listData, DOM.whatToDraw);
 
     const listViewButton = document.getElementById("listView");
     const cardViewButton = document.getElementById("cardView");
     const tableViewButton = document.getElementById("tableView");
-    const searchOperation = document.getElementById("searchOperation");
-
+    const searchOperation = document.getElementById("searchValue");
+    const showSunroof = document.getElementById("showSunroof");
+    const sunRoof = document.getElementById("sunRoof");
+    
     listViewButton.addEventListener("click", function () {
+        showSunroof.classList.add('d-none')
         DOM.whatToDraw = "list";
         draw(cars, DOM.listData, "list")
     })
     cardViewButton.addEventListener("click", function () {
+        showSunroof.classList.add('d-none')
```

Comment:
why do you need this?

Reply:
to switch between bootstrap classes, for the checkbox, that will appear only in table view.